lib/docs/src/checkbox.stories.tsx

```diff
@@ -1,27 +1,106 @@
-import { Meta } from "@storybook/react";
-import { Checkbox, DivPx } from "../../core/src";
+import { useState } from "react";
+import { DivPx, Checkbox } from "../../core/src";
+import { Utils } from "./utils";
 
 export default {
-	title: "Draft/Checkbox",
+	title: "Components/Checkbox",
 	component: Checkbox,
-} as Meta;
-
-export const Primary = (): JSX.Element => (
-	<div>
-		<Checkbox indeterminate>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox defaultChecked>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox>Checkbox</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled indeterminate>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled defaultChecked>
-			Checkbox
-		</Checkbox>
-		<DivPx size={8} />
-		<Checkbox disabled>Checkbox</Checkbox>
-	</div>
+	argTypes: {
+		children: Utils.arg(null),
+		checked: Utils.arg("boolean"),
+		indeterminate: Utils.arg("boolean"),
+		disabled: Utils.arg("boolean"),
+		setChecked: Utils.arg(null),
+		defaultChecked: Utils.arg(null),
+		forwardedRef: Utils.arg(null),
+	},
+};
+
+interface Props {
+	indeterminate: boolean;
+	disabled: boolean;
+	checked: boolean;
+}
+
+export const Primary = (props: Props): JSX.Element => {
+	return (
+		<>
+			<Checkbox
+				defaultChecked
+				checked={props.checked}
+				disabled={props.disabled}
+				indeterminate={props.indeterminate}
+			>
+				Checkbox
+			</Checkbox>
+			<DivPx size={8} />
+		</>
+	);
+};
+
+Utils.fixPrimary(Primary);
+
+type Book = {
+	id: string;
+	name: string;
+	author: string;
+};
+
+export const Group = (): JSX.Element => {
+	const books: Book[] = [
+		{
+			id: "tokillamockingbird",
+			name: "To Kill a Mockingbird",
+			author: "Harper Lee",
+		},
+		{
+			id: "thegreatgatsby",
+			name: "The Great Gatsby",
+			author: "F. Scott Fitzgerald",
+		},
+		{
+			id: "prideandprejudice",
+			name: "Pride and Prejudice",
+			author: "Jane Austen",
+		},
+	];
+	const [selected, setSelected] = useState<string[]>([]);
+
+	function toggleSelect(id: string) {
+		const newArray = selected.slice();
+		const isSelected = selected.includes(id);
+		if (!isSelected) {
+			newArray.push(id);
+		} else {
+			newArray.splice(newArray.indexOf(id), 1);
+		}
+		setSelected(newArray);
+	}
```

Comment:
I'm not really sure why this doesn't work in the deployment. Please check again.

Also, our target for the docs is ES6, so you can use the more modern alternatives to these. For example:

```ts
const isSelected = selected.includes(id);
if (isSelected) {
  setSelected(selected.filter(...));
} else {
  setSelected([...selected, id]);
}
```

But this is optional (and quite a personal thing). Just check to make sure it works.